core/src/main/scala/org/http4s/headers/ETag.scala

```diff
@@ -11,10 +12,14 @@ object ETag extends HeaderKey.Internal[ETag] with HeaderKey.Singleton {
       else "\"" + tag + '"'
   }
 
+  implicit val http4sShowForEntityTag: Show[EntityTag] =
```

Comment:
In the long run, we'll want to get this back into the companion object, so it's not an orphan instance.  But this is okay for now.